### PR TITLE
Start massaging Workflow into docs

### PR DESCRIPTION
**Pull request by slack**
Doing this piecemeal rather than all at once. Refs deis/docs-v2#7

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# Welcome
+# Welcome to Deis Workflow
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon Kubernetes to provide a lightweight PaaS with a Heroku-inspired workflow.
+Deis (pronounced DAY-iss) Workflow is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis Workflow adds a devleoper-friendly Platform as a Service (PaaS) layer to any Kubernetes cluster. Deis Workflow provides a simple REST API, application log aggregation, release and rollback, edge routing, source to image, and authentication and authorization.
```
Comment (mboersma) [inline src/index.md]: Typo: "developer-friendly"

Also "source to image" didn't flow when I read this. Could we say "source code to Docker image"?

File: src/index.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
-To get started with Deis, first read the [Architecture][arch] and [Concepts][concepts] of Deis, then use [the QuickStart documentation][quickstart] to start provisioning your cluster.
+To get started with Deis Workflow follow our [Quick Start][quickstart] guide.
```
Comment (carmstrong) [inline src/index.md]: `Deis Workflow, follow`

File: src/index.md (hunk 1 above)
Comment (carmstrong) [inline src/index.md]: Do we want to repeat `Deis Workflow` three times here? It reads a bit awkwardly. `It provides...` reads a little better.

File: src/installing-deis/installing-deis-workflow.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+```
+$ helm update
+$ helm repo add deis https://github.com/deis/charts
+$ helm fetch deis/deis
+```
+
+## Install Deis Workflow
+
+Now that you have Helm installed an the Deish Chart Repository added, install Workflow by running:
```
Comment (carmstrong) [inline src/installing-deis/installing-deis-workflow.md]: `Deish`

**Comment by carmstrong** [comment]
Looks good barring my comments.

File: src/installing-deis/installing-deis-workflow.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+We will use the `helm` utility to install Deis Workflow onto a Kubernetes cluster. If you don't
+have `helm` installed, see [installing helm][helm] for more info.
+
+## Check Your Setup
+
+First check that you have `helm` installed and the version is correct.
+
+    $ helm --version
+    0.2.0
```
Comment (mboersma) [inline src/installing-deis/installing-deis-workflow.md]: Deis beta will require `helm` 0.4.0 I believe.

File: src/index.md (hunk 1 above)
Comment (slack) [inline src/index.md]: Hit this with a hammer, still not satisfied.

File: src/installing-deis/installing-deis-workflow.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+$ helm target
+Kubernetes master is running at https://10.245.1.2
+Heapster is running at https://10.245.1.2/api/v1/proxy/namespaces/kube-system/services/heapster
+KubeDNS is running at https://10.245.1.2/api/v1/proxy/namespaces/kube-system/services/kube-dns
+KubeUI is running at https://10.245.1.2/api/v1/proxy/namespaces/kube-system/services/kube-ui
+Grafana is running at https://10.245.1.2/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana
+InfluxDB is running at https://10.245.1.2/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb
+```
+
+If you see a list of targets like the one above, helm can communicate with the kubernetes master.
```
Comment (mboersma) [inline src/installing-deis/installing-deis-workflow.md]: This should backtick `helm` to be consistent.

File: src/installing-deis/installing-deis-workflow.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+```
+$ helm install deis/deis --namespace=deis
+```
+
+Helm will install a variety of Kubernetes resources in the `deis` namespace.
+You'll need to wait for the pods that it launched to be ready. Monitor their status
+by running:
+
+```
+$ kubectl get pods --namespace=deis
```
Comment (mboersma) [inline src/installing-deis/installing-deis-workflow.md]: Should we add `--watch` / `-w` to this command?

Comment (slack) [inline src/installing-deis/installing-deis-workflow.md]: Added a `-w` section with additional help!

**Pull request merged**